packages/block-editor/src/layouts/flex.js

```diff
@@ -39,6 +52,12 @@ export default {
 					layout={ layout }
 					onChange={ onChange }
 				/>
+				{ layout?.orientation && (
```

Comment:
This check is interesting. It basically means that if the current layout doesn't have any orientation, do not display the control to switch "orientation". That said one can argue that the "no orientation" is just equivalent to "horizontal orientation" meaning we could just always show the toggle.

The problem with always showing the toggle is that it will show up in other blocks using the "flex" layout like the "Group" block (or its Row variation and potentially others). While one could argue that this is wrong since is a "row" right, conceptually it's not wrong. For instance, what about blocks like "Social Icons" and "Query Pagination" also using the "flex" layout, should we allow them to switch orientation? It might make sense.

So the thing here is that I'm hesitant between:

 - Keep things as they are in the PR: making the "orientation" undefined or not in the default layout value the actual flag that indicates whether to show this control or not.
 - Introduce a dedicated opt-out flag in the layout support config (and potentially only use it in the Group block)

Reply:
An opt-out flag would be nice, especially if there's only one block where we really don't want to enable vertical orientation. Perhaps something like `"allowOrientation": false`?

It also occurred to me that if orientation were enabled for all blocks, we wouldn't need Row as a variation of Group at all. But Group is using flow layout; would there be any issues in migrating it to flex with vertical orientation?

Reply:
Well, a flexbox layout doesn't work with `float`s, so yes, there would definitely be issues if "flow" did not remain as an option.

Reply:
Oooh good point. I guess we'd better leave it as it is then.